### PR TITLE
Consensus account details cards

### PR DESCRIPTION
**Pull request by buberdds**
- Balance distribution card
- Staking card

### CODE REVIEW

File: src/oasis-nexus/api.ts [hunk 1]
```diff
@@ -949,3 +955,34 @@ export const useGetConsensusAccounts: typeof generated.useGetConsensusAccounts =
...
+        transformResponse: [
+          ...arrayify(axios.defaults.transformResponse),
+          (data: generated.DelegationList, headers, status) => {
+            if (status !== 200) return data
+            return {
+              ...data,
+              delegations: data.delegations.map(delegation => {
+                return {
+                  ...delegation,
+                  shares: fromBaseUnits(delegation.shares, consensusDecimals),
```
Comment (lukaw3d) [inline src/oasis-nexus/api.ts:975]: Shares don't have decimals like this. I don't know if we want to convert them and use "gigashares" throughout the app

File: src/app/pages/ConsensusAccountDetailsPage/Staking.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  const scope = useRequiredScopeParam()
+  const { network } = scope
+  const delegationsQuery = useGetConsensusAccountsAddressDelegations(
+    network,
+    address,
+    {},
+    {
+      query: {
+        cacheTime: 0,
+      },
```
Comment (lukaw3d) [inline src/app/pages/ConsensusAccountDetailsPage/Staking.tsx:68]: How come cacheTime is 0 here? I thought we were just using that for a few "latest" pages to trigger loading states (https://github.com/oasisprotocol/explorer/pull/732)

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/ConsensusAccountDetailsPage/Staking.tsx (hunk 2 above)
Comment (buberdds) [reply src/app/pages/ConsensusAccountDetailsPage/Staking.tsx]: I have clean this up and it is used only in Latest* cards now.

File: src/oasis-nexus/api.ts (hunk 1 above)
Comment (buberdds) [reply src/oasis-nexus/api.ts:975]: For now I will merge as is to avoid conflicts in PR and will address this in a next PR if needed.

Comment (lukaw3d) [reply src/oasis-nexus/api.ts:975]: Notes from discussion:

Out of 
- 213456546455 shares
- 213.456546455 gigashares
- 213.45... B shares
- 213B shares

"213B shares" is probably nicest

We could reuse the code from token supply display https://explorer.dev.oasis.io/mainnet/sapphire/token ; decimals are not very important
image

Comment (buberdds) [reply src/oasis-nexus/api.ts:975]: relates to https://github.com/oasisprotocol/explorer/pull/1310

Comment (buberdds) [reply src/oasis-nexus/api.ts:975]: Can you verify this in #1310 please?
sample acc: https://0a9444e9.oasis-explorer.pages.dev/mainnet/consensus/address/oasis1qqnk4au603zs94k0d0n7c0hkx8t4p6r87s60axru 
sample validator (with hardcoded GoStaking entity) https://0a9444e9.oasis-explorer.pages.dev/mainnet/consensus/validators/oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v/delegators#delegators

**Pull request merged**